src/packages/core/src/server.ts

```diff
@@ -183,9 +183,7 @@ export class Server extends Emittery<{ open: undefined; close: undefined }> {
           else throw err;
         }
       })
-    ]).then(() => {
-      return this.emit("open");
```

Comment:
I put this in the wrong spot during rebase; this change isn't one that is in `develop`. The `"open"` event was added in https://github.com/trufflesuite/ganache-core/pull/847/files#diff-b4ab586220123bb3565eb84b69a47164e6012bd51caea50e24bbe8b89879adfcR184